cli/cmd/initnodes.go

```diff
@@ -442,6 +442,29 @@ func gethInit(ctx context.Context, cfg InitConfig, dir string) error {
 	return nil
 }
 
+func downloadSnapshots(ctx context.Context, cfg InitConfig) error {
```

Comment:
rename to `maybeDownloadSnapshots` since it doesn't always do this.

Also, avoid nesting, rather do:
```
if !cfg.NodeSnapshot {
  return nil
}

...
```